pkg/controller/issueapi/issue.go

```diff
@@ -80,6 +84,7 @@ func (c *Controller) HandleIssue() http.Handler {
 		var request api.IssueCodeRequest
 		if err := controller.BindJSON(w, r, &request); err != nil {
 			c.h.RenderJSON(w, http.StatusBadRequest, api.Error(err))
+			stats.Record(ctx, c.metrics.CodeIssueErrors.M(1))
```

Comment:
Probably doesn't matter, but there's some inconsistencies with this - sometimes we stat -> render, sometimes we render -> stat